projects/igniteui-angular/src/lib/date-picker/date-picker.component.ts

```diff
@@ -549,9 +550,16 @@ export class IgxDatePickerComponent implements IDatePicker, ControlValueAccessor
      *<igx-date-picker (onOpen)="open($event)" cancelButtonLabel="cancel" todayButtonLabel="today"></igx-date-picker>
      *```
      */
+    @DeprecateProperty(`'onOpen' @Ouput property is deprecated\nUse 'onOpened' instead.`)
```

Comment:
Typo `@Ouput`. Replace `\n` with `.` so there won't be a new line in the output.